packages/i18n/.eslintrc.json

```
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "es2021": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
```

You don't need the `"prettier"` 🙂. [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) serves only to turn off ESLint rules that conflict with formatters such as Prettier... but neither `"eslint:recommended"` nor `"plugin:@typescript-eslint/recommended"` enable any like that anymore.

```suggestion
    "plugin:@typescript-eslint/recommended"
```

That's not to be confused with [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier), which _runs_ Prettier within an ESLint rule. Very different thing.

---

Yeah, we def do not want to run prettier under ESLint.